### ISSUE TITLE
Add support for the binomial deviance loss function

### ISSUE THREAD

**Issue by vruusmann** [jpmml/jpmml-sklearn#103]
As reported here: https://github.com/openscoring/openscoring/issues/49

Apparently, GitHub doesn't allow migrating issues from one organization (openscoring) to another (jpmml), hence opening  new issue here.

**Comment by sam-s** [comment]
Same error for exponential deviance - the problem is, apparently, an internal reorg of sklearn class hierarchy.

**Comment by vruusmann** [comment]
> the problem is, apparently, an internal reorg of sklearn class hierarchy.

The mapping is defined in the sklearn2pmml.properties file:
https://github.com/jpmml/jpmml-sklearn/blob/master/src/main/resources/META-INF/sklearn2pmml.properties#L37-L47

I've been re-building my integration test pickle files after every major Scikit-Learn package update. Need to maintain old mappings for backward compatibility, and add new mappings to whatever the new Python module name is.

BTW, what's your Scikit-Learn version that exhibits this problem?

**Comment by sam-s** [comment]
`sklearn.__version__ == '0.21.0'`

**Comment by sam-s** [comment]
Same problem after `pip3 install --upgrade 'git+https://github.com/jpmml/sklearn2pmml.git#egg=sklearn2pmml'`
Please advise.
